File: 프로그래머스/1/12935. 제일 작은 수 제거하기/제일 작은 수 제거하기.py
def solution(arr):
    if len(arr) == 1:
        return [-1]
    
    min_value = min(arr)  # 최소값 찾기
    arr.remove(min_value)  # 리스트에서 최소값 제거 (첫 번째 값만 제거됨)
    return arr

File: 프로그래머스/1/12935. 제일 작은 수 제거하기/제일 작은 수 제거하기.kt
class Solution {
    fun solution(arr: IntArray): IntArray {
        // 배열의 크기가 1이면 [-1] 반환
        if (arr.size == 1) return intArrayOf(-1)
        
        // 최소값 찾기
        val minValue = arr.minOrNull()!!
        
        // 최소값 제거하고 새로운 배열 생성
        return arr.filter { it != minValue }.toIntArray()
    }
}
